### ISSUE TITLE
I'm being recommended sites I've never visited

### ISSUE THREAD

**Issue by folz** [mozilla/universal-search#306]
TL;DR this issue's subject. I don't remember this happening in the past while using this addon.

![recommended-sites](https://cloud.githubusercontent.com/assets/299281/18036515/15f32c2c-6d21-11e6-89f6-974d50ff2716.png)

In this screenshot, I have typed "mem" and intend to navigate to members.nearlyfreespeech.net. Instead, I see memegenerator.net as the first result. I've never been to this site and have no intention or desire to visit it.

The screenshot doesn't show it, but when navigating through this list, the recommended site sometimes pops up after a delay of 1-2 seconds and resets the selection position, meaning I load the recommended site even though I'm trying to open a different site.

I would like the option to disable recommendations entirely, but would at the very least prefer being recommended sites I've already visited. In the meantime, I set the "recommendation server" option to a (nonexistent) local server, which seems to have stopped the recommendations.

**Comment by jaredhirsch** [comment]
@folz Hey there - the point of this particular experimental add-on is to show recommendations for things you don't have in history. (For example, in a fresh profile, typing 'f' will get you a recommendation for Facebook.) Note that, unlike the vast majority of recommendations on the web, we generate those recommendations in a user privacy-preserving manner. But yeah, if you don't want recommendations, you should uninstall the add-on.

Summing up: add-on is working as intended. Closing

**Comment by folz** [comment]
Thanks for the detailed response to my issue. When I originally saw Universal Search, I thought its purpose was an [omnibar](https://addons.mozilla.org/en-US/firefox/addon/omnibar/)-style addon that happened to have recommendations, not a recommendations engine that happened to look like omnibar. I appreciate you clearing up my confusion :)
